docs/extra_pages.dox

```diff
@@ -0,0 +1,177 @@
+/** \mainpage
+ *
+ * **easyRNG** is a simple library for C and Fortran providing access to several pseudo-random
```

Comment:
Is het de bedoeling dat dit aangepast wordt naar polycap informatie, of is dit iets om te houden gezien de EasyRNG library gebruikt wordt?

Reply:
Dit is de tekst die zal komen op de main page van de website: de index.html eigenlijk.

Hier moet dus een introductie tekst komen over polycap.

Reply:
Okiedo, dacht al zoveel. Maar deze tekst mag dus weg in principe he? Of moet er nog ergens uitgelegd worden wat EasyRNG is ook?

Reply:
Wel niet op de main page denk ik... misschien beter bij de installation instructions, aangezien het een optionele dependency is.

Reply:
👍